plugins/modules/host.py

```diff
@@ -225,6 +381,39 @@ def main():
             provision_method=dict(choices=['build', 'image', 'bootdisk']),
             image=dict(type='entity', scope=['compute_resource']),
             compute_attributes=dict(type='dict'),
+            interfaces_attributes=dict(type='list', elements='dict', options=dict(
+                mac=dict(),
+                ip=dict(),
+                ip6=dict(),
+                type=dict(default='interface', choices=['interface', 'bmc', 'bond', 'bridge']),
+                name=dict(),
+                subnet_id=dict(type='int'),
+                subnet6_id=dict(type='int'),
+                domain_id=dict(type='int'),
+                identifier=dict(),
+                managed=dict(type='bool'),
+                primary=dict(type='bool'),
+                provision=dict(type='bool'),
+                username=dict(),
+                password=dict(no_log=True),
+                provider=dict(choices=['IPMI', 'SSH']),
+                virtual=dict(type='bool'),
+                tag=dict(),
+                mtu=dict(type='int'),
+                attached_to=dict(),
+                mode=dict(choices=[
+                    'balance-rr',
+                    'active-backup',
+                    'balance-xor',
+                    'broadcast',
+                    '802.3ad',
+                    'balance-tlb',
+                    'balance-alb',
+                ]),
+                attached_devices=dict(type='list'),
```

Comment:
the sanity tests complain:
` Argument 'attached_devices' in argument_spec found in interfaces_attributes defines type as list but elements is not defined`

I think adding `elements='str'` here is sufficient